app/views/playlist.js

```diff
@@ -3,8 +3,18 @@ import application from '../application'
 
 const PlaylistView = Mn.ItemView.extend({
 
+  template: require('./templates/popupPlaylists'),
+
   tagName: 'li',
 
+  ui: {
+    'play': '.play'
+  },
+
+  events: {
+    'dblclick': 'playCurrentPlaylist'
```

Comment:
Is the double-click action occurs on the whole wrapper tag (aka `li` or on a dedicated element (like `.play`)?